### ISSUE TITLE
Epic: Self-Hosted via KOTS/Replicated  As the default "Skateboard"

### ISSUE THREAD

**Issue by lucasvaltl** [gitpod-io/gitpod#9072]
### Summary

We want to make the Kubernetes Off-The-Shelf (KOTS, a.k.a replicated) installation path the default for all self-hosted users.

### Context

In order to give self hosted customers a better installation, upgrade and licensing management experience, we want to leverage KOTS, or rather Replicated. This means that we (Gitpod) do not need to do all of the heavy lifting ourselves when it comes to servicing self-hosted customers, and can instead rely on Replicated.

We've already done considerable work in this area(#8540), and as such this Epic is targeted at making Replicated the default installation experience for all self-hosted users and customers. 

### Value

- Facilitates onboarding and operation of self-hosted Gitpod instances
- Facilitates license management for Gitpod
- Allows for opting-in to different release channels

### Acceptance Criteria

- Our website and documentation is updated to reflect Replicated as the default path
- Licensing is smooth and straight forward
- [x] #9041
- [x] #8905
- [x] #8906 
- [ ] #8840 
- [ ] #8789 (currently blocked by [external issue](https://github.com/replicated-collab/replicated-gitpod/issues/4))
- [x] #9174 
- [x] #8883 
- [x] #9074
- [ ] #3918
- [x] #9562
- [ ] #9820 
- [ ] #9926 
- More criteria TBD and will depend on the outcome of user testing of the Replicated Beta

**Comment by lucasvaltl** [comment]
Closing this and moving open issues to #9929 - the goal here of making replicated the default has been reached. We will further solidify and improve on the installation experience via #9929.
